### DISCUSSION TITLE
Coin Request: VerusCoin and Ravancoin (KAWPOW)

### DISCUSSION THREAD

**Discussion by BrianBMN** [coinfoundry/miningcore#921] [Ideas]
I'd like to see VerusCoin added and Ravencoin updated to KAWPOW. If I was a developer I would take this but alas I am not. I can however test and debug like a ninja!

**Comment by bdk773** [comment]
Would love to see kawpow as well!

**Comment by calvintam236** [comment]
See #876 for Ravencoin

**Comment by belmix** [comment]
VerusCoin is implemented at the very least on the old release (add algo VerusHash)
https://github.com/belmix/miningcore/commits/master
